buildtest/cli/buildspec.py

```diff
@@ -874,6 +874,7 @@ def print_invalid_buildspecs(self, error=None, terse=None, header=None):
 
         if error and terse:
             console.print("The --terse flag can not be used with the --error option")
+            sys.exit(1)
             return
```

Comment:
we can remove this `return` statement since this statement will never be reached